docs_src/getting_started.md

```diff
@@ -0,0 +1,76 @@
+# Getting Started
+
+## Step by step instructions:
+
+1. Download the models using download_models/downloadModels.sh
+```bash
+make download-models
+```
+
+2. Update github submodules
+```bash
+make update-submodules
+```
+
+3. Download sample videos used by the performance tools
+```bash
+make download-sample-videos
+```
+
+4. Build the demo Docker image
+```bash
+make build
+```
+
+5. Start Automated Self Checkout using the Docker Compose file. The Docker Compose also includes an RTSP camera simulator that will infinitely loop through the sample videos downloaded in step 3.
+```bash
+make run-render-mode
+```
+
```

Comment:
can you add an image from portainer showing all the containers that start after running this command, just so that user can validate everything is working. You can also add a screenshot from the video with bounding boxes.

Reply:
That's a great idea, I'll add it in the verify results section.